src/components/layouts/collection-page-layout.tsx

```diff
@@ -559,20 +624,39 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
                     >
                       <div
                         className="text-gray-600 dark:text-gray-300 flex flex-row items-center rounded hover:bg-gray-200 
-                      dark:hover:bg-gray-700 border-gray-300 bg-white dark:bg-gray-800 dark:border-gray-600 px-4 py-2 border transition-colors text-sm xs:text-base ease-in-out"
+                      dark:hover:bg-gray-700 border-gray-300 bg-white dark:bg-gray-800 dark:border-gray-600 px-4 py-2 border transition-colors text-sm xs:text-base ease-in-out shadow-sm"
                       >
                         <FolderDownloadIcon className="w-4 h-4 mr-1" /> Download
                       </div>
                     </a>
                   </Link>
                 ) : (
-                  <div
-                    className="text-gray-600 dark:text-gray-300 flex flex-row items-center rounded hover:bg-gray-200 
-                  dark:hover:bg-gray-700 border-gray-300 bg-white dark:bg-gray-800 dark:border-gray-600 px-4 py-2 border transition-colors text-sm xs:text-base ease-in-out opacity-30"
+                  <DialogButton
+                    buttonText="Download"
+                    title="Become a Pro member to download this course"
+                    buttonStyles="text-gray-600 dark:text-gray-300 flex flex-row items-center rounded hover:bg-gray-100 
+                      dark:hover:bg-gray-700 border-gray-300 bg-white dark:bg-gray-800 dark:border-gray-600 px-4 py-2 border transition-colors text-sm xs:text-base ease-in-out opacity-90 shadow-sm"
                   >
-                    <FolderDownloadIcon className="w-4 h-4 mr-1" /> Download
-                  </div>
+                    <div className="flex flex-col space-y-4">
+                      <p className="max-w-10 text-center text-gray-700">
+                        As a member you can download any of our courses and
+                        watch them offline. As a member you'll also get:
+                      </p>
+                      <PlanFeatures />
+
+                      <button className="font-semibold w-full mt-6 inline-flex justify-center items-center px-4 py-3 rounded-md bg-blue-600 text-white transition-all hover:bg-blue-700 ease-in-out duration-200">
+                        Become a Member
+                      </button>
+                      <Link href="/pricing">
```

Comment:
suggestion: this link (if used now) should likely go to the `/about` page, but that page isn't great and I'd just drop this until it's better